### ISSUE TITLE
New Tasks JS error and not adding.

### ISSUE THREAD

**Issue by thebeline** [backlogs/redmine_backlogs#914]
platform:  # supported: 2.2.4, 2.3.0
backlogs:  # supported: 0.9.38
ruby:  # supported: 1.9.3, 2.0.0

**Versions:**
- Environment:
  - Redmine version                          2.3.0.stable
  - Ruby version                             1.8.7 (x86_64-linux)
  - Rails version                            3.2.13
  - Environment                              production
  - Database adapter                         Mysql2
- Redmine plugins:
  - redmine_backlogs                         v0.9.38
  - redmine_import_from_csv                  0.0.1

I go to `...com/rb/taskboards/2` and click the plus for new task.

The new task dialog appears at T0/L0 (annoying, to say the least) with a translucent/transparent background (also annoying, to say the least).  I can enter text and click 'OK'. The resulting dialog that shows up has holders for text:

> New Task
> Subject
> Description
> Assignee
> Remaining (Hours)

But My data is not there.

Clicking `OK` again does nothing.  Only `CANCEL` does anything.

Clicking `OK` throws this error:
Uncaught TypeError: Cannot call method 'split' of undefined

Basically, Line 69 of assets/javascript/task.js: `var cellID = j.parents('td').first().attr('id').split("_");` is not working, everything before the split is not returning a valid object to call split on.  I would debug it, but the program flow is difficult to follow at this time.

Really, I don't expect it to work if it can't even show me what I entered.

On that actually, no errors are thrown that would indicate why my data doesn't propagate to the confirmation dialog.
